homework01/artur/vector.h

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+class Vector
+{
+ public:
+  Vector();
+  Vector(const int, const int);
+  Vector(const Vector&);
+  
+  const int GetX() const;
+  const int GetY() const;
+
+  const bool operator==(const Vector&);
+  const Vector operator+(const Vector&);
```

Comment:
Here as the returned value is an object, `const` hurts. We will discuss about it Thursday